kratos/python_scripts/testing_utilities.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import print_function, absolute_import, division
+
+# Import Kratos
+import KratosMultiphysics
+try:
+    import KratosMultiphysics.mpi as KratosMPI
+    rank = KratosMPI.mpi.rank
```

Comment:
Is this safe? I tried this out in interactive python and I get an error code 2 which apparently means MPI_ERR_COUNT.

Reply:
I thought it works
At least I use sth similar in [the MappingTests](https://github.com/KratosMultiphysics/Kratos/blob/master/applications/MappingApplication/tests/KratosExecuteMapperTests.py#L35-L46) for a long time now

I still removed it bcs I put the function to the unittest and I didn't want to introduce the dependencie on mpi. If it is there at some point we can add it